app/javascript/packs/application.js

```diff
@@ -2,25 +2,56 @@
 // present in this directory. You're encouraged to place your actual application logic in
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
-import 'bootstrap'
-import './src/application.scss'
+import "bootstrap";
+import "./src/application.scss";
 
-require("@rails/ujs").start()
-require("turbolinks").start()
-require("@rails/activestorage").start()
-require("channels")
+require("@rails/ujs").start();
+require("turbolinks").start();
+require("@rails/activestorage").start();
+require("channels");
+require("jquery");
 
 document.addEventListener("turbolinks:load", () => {
-  $('[data-toggle="tooltip"]').tooltip()
-  $('[data-toggle="popover"]').popover()
-  $('.toast').toast({ autohide: false })
-  $('#toast').toast('show')
-})
+  $('[data-toggle="tooltip"]').tooltip();
+  $('[data-toggle="popover"]').popover();
+  $(".toast").toast({ autohide: false });
+  $("#toast").toast("show");
+});
 
+$(document).ready(function() {
+  $(".showCommentbtn").click(function() {
+    console.log("CALLED showCommentbtn chk --0");
```

Comment:
Before pushing the code always check if there are `console.log()` to be removed. 
(by using`CMD+F`)